export const currentUser = async (req , res) => {
    try {
        
    } catch (error) {
        
    }
}